Vizzuality/half-earth-v3
# feat: translate onboarding test [HE-392]

## Translate onBoarding test

### Description
Translate SCRPITS strings.
Add test to first step of Priority Places onBoarding in spanish and portuguese.
Remove Esperanto as language option.

### Testing instructions
Please set spanish in the home page and go to pririty places on boarding, check the scripts are in spanish and there is enough space on the dialog to display the texts.
Then go back home and select portuguese, go again to priority places onboarding >> There is an ERROR here, as scripts will be still in spanish but "Analyze Areas" text on left has been already translate to portuguese.

### Feature relevant tickets
[HE-392](https://vizzuality.atlassian.net/browse/HE-392)
